arm/authorization/client.go

```diff
@@ -0,0 +1,759 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 0.11.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package authorization
+
+import (
+	"fmt"
+	"github.com/azure/go-autorest/autorest"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+const (
+	ApiVersion             = "2015-01-01"
+	DefaultBaseUri         = "https://management.azure.com"
+	DefaultPollingDuration = 10 * time.Minute
+)
+
+type AuthorizationClient struct {
+	autorest.Client
+	BaseUri        string
+	SubscriptionId string
+}
+
+func New(subscriptionId string) *AuthorizationClient {
+	return NewWithBaseUri(DefaultBaseUri, subscriptionId)
+}
+
+func NewWithBaseUri(baseUri string, subscriptionId string) *AuthorizationClient {
+	client := &AuthorizationClient{BaseUri: baseUri, SubscriptionId: subscriptionId}
+	client.PollingMode = autorest.PollUntilDuration
+	client.PollingDuration = DefaultPollingDuration
+	return client
+}
+
+////////////////////////////////////////////////////////////////////////////////
+//
+// ManagementLocks Client
+//
+////////////////////////////////////////////////////////////////////////////////
+type ManagementLocksClient struct {
+	AuthorizationClient
+}
+
+func NewManagementLocksClient(subscriptionId string) *ManagementLocksClient {
+	return NewManagementLocksClientWithBaseUri(DefaultBaseUri, subscriptionId)
+}
+
+func NewManagementLocksClientWithBaseUri(baseUri string, subscriptionId string) *ManagementLocksClient {
+	return &ManagementLocksClient{*NewWithBaseUri(baseUri, subscriptionId)}
+}
+
+// CreateOrUpdateAtResourceGroupLevel create or update a management lock at
+// the resource group level.;
+//
+// resourceGroupName is the resource group name. lockName is the lock name.
+// parameters is the management lock parameters.
+func (client *ManagementLocksClient) CreateOrUpdateAtResourceGroupLevel(resourceGroupName string, lockName string, parameters ManagementLock) (result ManagementLock, err error) {
+
```

Comment:
I'm saying it would be nice to remove these empty lines; not add:

> ![image](https://cloud.githubusercontent.com/assets/159209/9262107/47cad122-41c7-11e5-9f74-8070a2003525.png)

> 
![image](https://cloud.githubusercontent.com/assets/159209/9262116/4f68dcee-41c7-11e5-8bcd-c42b89e4d783.png)